checker/src/main/java/org/checkerframework/checker/lock/qual/EnsuresLockHeldIfMultiple.java

```diff
@@ -0,0 +1,34 @@
+package org.checkerframework.checker.lock.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.ConditionalPostconditionAnnotation;
+import org.checkerframework.framework.qual.InheritedAnnotation;
+
+/**
+ * An annotation wrapper to allow multiple postcondition annotations.
+ *
+ * <p>Programmers generally do not need to use this; it is created by Java when a programmer writes
+ * more than one {@code @EnsuresLockHeldIf} annotation at the same location.
+ *
+ * @see EnsuresLockHeld
+ * @checker_framework.manual #lock-checker Lock Checker
+ * @checker_framework.manual #ensureslockheld-examples Example use of @EnsuresLockHeldIf
+ */
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.CONSTRUCTOR})
+@ConditionalPostconditionAnnotation(qualifier = LockHeld.class)
+@InheritedAnnotation
+public @interface EnsuresLockHeldIfMultiple {
+    /**
+     * Constitutes the java expressions whose values are held after the method returns the given
+     * result.
+     *
```

Comment:
This doesn't have the text about how programmers don't need to write the annotation.  I'm fine with omitting it, but then do so throughout, so that all the new annotations are consistent.